adserver/views.py

```diff
@@ -1040,6 +1041,7 @@ class AdvertiserReportView(AdvertiserAccessMixin, BaseReportView):
     """A report for one advertiser."""
 
     export_view = "advertiser_report_export"
+    report = AdvertiserReport
```

Comment:
Wonder what it would take to use the optimized index here? We could have a seperate non-default report if you wanted to see non-paid ads to make this faster? 🤔

Though I think things like fill rate might get busted with this.

Reply:
It might, but I don't think it's necessary. Doing the slower report for a single advertiser should be plenty fast.

Reply:
Yea, except when they have a ton of data (eg. RTD or some of our largest advertisers over a 3+ month time period)

Reply:
Probably not a bad idea to implement it then.